CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Plugin supports validation of self-signed certificates provided in `CONJUR_CERT_FILE`
   or Conjur config file
   ([cyberark/ansible-conjur-collection#4](https://github.com/cyberark/ansible-conjur-collection/issues/4))
+- Plugin support authenticating with access token provided by authn-k8s
```

Comment:
once there is an issue filed, it would be good to link to it here. also, this entry should be moved to a new `### Added` section under `Unreleased`